src/IssueViz.UnitTests/Editor/LocationNavigatorTests.cs

```diff
@@ -68,116 +76,163 @@ public void MefCtor_CheckIsExported()
         }
 
         [TestMethod]
-        public void TryNavigate_LocationIsNull_DocumentNotOpened()
+        public void TryNavigate_LocationIsNull_ArgumentNullException()
         {
-            var result = testSubject.TryNavigate(null);
+            Action act = () => testSubject.TryNavigate(null);
 
-            result.Should().BeFalse();
+            act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("locationVisualization");
 
             documentOpenerMock.VerifyNoOtherCalls();
+            spanCalculatorMock.VerifyNoOtherCalls();
         }
 
         [TestMethod]
-        [DataRow(null)]
-        [DataRow("")]
-        public void TryNavigate_LocationHasNoFilePath_DocumentNotOpened(string filePath)
+        public void TryNavigate_FailedToOpenDocument_NoException()
```

Comment:
For me, it isn't clear from the name `FailedToX` that the failure was caused by an exception. `ErrorOpeningDocument_` is less ambiguous.